tensorflow_model_optimization/g3doc/guide/clustering/clustering_comprehensive_guide.ipynb

```diff
@@ -0,0 +1,536 @@
+{
```

Comment:
<p>"See "Define model" and "Train model" on navigation sidebar for how to define</p><p> and train this model in other ways."</p><p> </p><p>I've been told to not reference the navigation bar. You also don't have a "Train model" section. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/model-optimization/pull/330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>